.github/workflows/ci_cron_weekly.yml

```diff
@@ -29,64 +29,34 @@ env:
 
 jobs:
   tests:
-    runs-on: ${{ matrix.os }}
     if: (github.repository == 'astropy/astropy' && (github.event_name == 'schedule' || github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'Extra CI')))
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     env:
       ARCH_ON_CI: "normal"
```

Comment:
This don't go together. It is a syntax error. If you check the jobs for this PR, weekly cron jobs were missing. This was merged prematurely. See https://github.com/astropy/astropy/pull/14070